spacetemplate/assets/agile.yaml

```diff
@@ -166,9 +170,11 @@ work_item_types:
   extends: *agileCommonTypeID
   name: Story
   can_construct: yes
-  description: 
-    A user story describes an executable chunk of a feature 
-    based on user experience.
+  description:
+    Incremental work or requirements identified to deliver a product feature.
+    Typically, it is expressed from the user’s perspective in the following
```

Comment:
Can you please be careful with characters here? I don't think `user’s` is right but `user's` is.

Reply:
I've made the change to `'`.